src/Microsoft.OData.Core/Microsoft.OData.Core.csproj

```diff
@@ -1,14 +1,15 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net8.0</TargetFramework>
+    <TargetFramework>net10.0</TargetFramework>
     <AssemblyName>Microsoft.OData.Core</AssemblyName>
     <RootNamespace>Microsoft.OData.Core</RootNamespace>
 
     <DocumentationFile>$(AssemblyName).xml</DocumentationFile>
     <DefineConstants>$(DefineConstants);ODATA_CORE;SUPPRESS_PORTABLELIB_TARGETFRAMEWORK_ATTRIBUTE;DelaySignKeys</DefineConstants>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <TargetFrameworks>net10.0</TargetFrameworks>

```

Comment:
```suggestion
```
You can remove the other target frameworks from Build.props

Reply:
https://github.com/OData/odata.net/pull/3204/files#diff-2dde510688211f1479ac8bbd63641430561100e5ac6b5e50f6e8821a9322bfb9 nit